shell/edit/auth/oidc.vue

```diff
@@ -59,42 +68,67 @@ export default {
       };
     },
 
+    validationPassed() {
```

Comment:
This block could be tidied up, and we shouldn't lose the validation for keycloak